projects/plugins/jetpack/changelog/update-wp_register_block_metadata_collection

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: enhancement
+
+Use wp_register_block_metadata_collection() on WordPress 6.5+ to improve block registration performance by reducing filesystem operations. (See https://core.trac.wordpress.org/changeset/59132)
```

Comment:
Looks like the function has `@since 6.7.0`, not 6.5.
```suggestion
Use wp_register_block_metadata_collection() on WordPress 6.7+ to improve block registration performance by reducing filesystem operations. (See https://core.trac.wordpress.org/changeset/59132)
```

Reply:
Thanks, :facepalm: